README.md

```diff
@@ -92,6 +92,8 @@ Usage
             elasticsearch server, this works with https as well
       -size int
             bulk batch size (default 1000)
+      -sjson 
```

Comment:
"sjson" sound so, benign - I'd call it "-skip-broken", "-skip", "-best-effort" - these sound a bit more after the broken nature of the input.